index.html

```diff
@@ -41,8 +44,11 @@ <h2 class="flex justify-center">Please choose from the following list after sear
                 </div>
             </div><!--section for rendered recipe list-->
         </div>
-
-       
+              <!--multiple selected options-->
```

Comment:
This creates an odd box on the page, can we comment this code out until we figure out how to integrate it properly?